apps/discord-bot/src/commands/megawalls/megawalls.profile.tsx

```diff
@@ -98,3 +98,17 @@ export const MegaWallsProfile = ({
     </Container>
   );
 };
+
+export function filterMegaWallsKits(
+  player: Player,
+  modes: GameMode<MegaWallsModes>[]
+): GameMode<MegaWallsModes>[] {
+  const { megawalls } = player.stats;
+  const [overall, ...kits] = modes;
+
+  const filteredKits = [...kits]
+    .sort((a, b) => megawalls[b.api].points - megawalls[a.api].points)
+    .slice(0, 24);
```

Comment:
Is this 24 supposed to be the amount of kits ? If that's the case it's 27 now

Reply:
No. Discord has a dropdown limit of 25.